src/components/component.ts

```diff
@@ -236,9 +237,19 @@ export class Component {
     if (this._clipPathEnabled) {
       this._updateClipPath();
     }
+    this._renderImmediately();
```

Comment:
I think I meant for the above 3 lines to actually exist in `_renderImmediately`  I will adjust